docs/javascript-overview.md

```diff
@@ -47,4 +47,13 @@ The `interactive` object contains the following properties, corresponding to the
 * `interactive.csharp`
 * `interactive.fsharp`
 * `interactive.pwsh`
-* `interactive.value`
\ No newline at end of file
+* `interactive.value`
+
+## Loading external JavaScript modules at runtime
+
+Sometimes you might need to import JavaScript modules in your notebook. You can use the `interactive` object to do so.
+
+Here is an example.
+![image](https://user-images.githubusercontent.com/375556/95768858-e6be3500-0cae-11eb-925c-53d46670bd09.png)
```

Comment:
Let's use code rather than images to document APIs. This will be hard to search for otherwise.

Reply:
removed image and made test example that users can actually use and try